### ISSUE TITLE
Please supply better instructions. No render template displayed and errors occurring

### ISSUE THREAD

**Issue by jallwork** [alexa-samples/skill-sample-python-quiz-game#20]
I'm trying to follow the instructions, but they are not complete. I don't see the displayRender template appearing when using the developer console and I am also getting errors. Here's what I'm doing:

The instructions say: "to use this template, select Quiz Game Skill and click "Choose". 
The options are "Import skill" or "Continue with template"
If I try to Import skill using clone git https://github.com/alexa/skill-sample-python-quiz-game.git
I get the error: There was an issue creating your skill. Skill resources were not provisioned. Please try again later.
So I "Continue with template"
I run the program - the voice interface works, but I only see 'quiz game' displayed on the dev console display (using default Medium Hub) 
"Alexa Presentation Language" is enabled in the Interfaces and the skill has been rebuilt
Trying to look at the CloudWatch logs, gives me the error: There was an error loading Log Streams. Please try again by refreshing this page. 
Refreshing gives them same error.
I can't find the ARN in my CloudWatch log stream either (arn:aws:lambda:us-east-1:164765733365:function:ff248fa9-e18f-40f2-9729-b4721350bfa4:Release_0)

If I investigate this error, I get to
https://aws.amazon.com/premiumsupport/knowledge-center/lambda-cloudwatch-log-streams-error/
     Resolution - Make sure that your Lambda function's execution role has sufficient permissions to write logs to CloudWatch
- with more instructions - Do I have to do this?

**Comment by ekund** [comment]
I am having the same issue with the instructions.

**Comment by paoloantinori** [comment]
same. no ui rendering at all

**Comment by Hugemege** [comment]
I´ve got the same issue, please help.
Try to import to alexa skills: There was an issue creating your skill. Skill resources were not provisioned. Please try again later.

**Comment by aszk** [comment]
I believe there are couple of issues are reported here.
1. Unable to import to Alexa-hosted
This will be fixed once my PR is merged.
 
2. Rendering issue. (Displaying skill name)
This is expected behavior. This sample code doesn't include APL document. Please refer https://github.com/alexa-samples/skill-sample-python-sauce-boss or https://developer.amazon.com/en-US/docs/alexa/alexa-presentation-language/create-and-edit-apl-document.html. I'd recommend using authoring tool. 

3. An issue to write CloudWatch logs
Thank you @jallwork! If you use AWS, you may need to add some permission to write logs to CloudWatch. If you use Alexa-hosted, you don't need to do extra work.

I am going to close this because we have another issue regarding 1. In addition, 2 and 3 were answered.
